setuptools/__init__.py

```diff
@@ -109,7 +109,10 @@ def _looks_like_package(path):
 
 find_packages = PackageFinder.find
 
-setup = distutils.core.setup
+def setup(**attrs):
+    if 'parse_config' not in attrs:
+        attrs['parse_config'] = True
+    distutils.core.setup(**attrs)
```

Comment:
I'm not sure what `distutils.core.setup()` return type is, but I'd prepend this line with `return `, just to be on the safe side.

Reply:
The resulting distribution object should be returned. And this behavior is actually tested, except the tests are not picked up by pytest (because they are in `setuptools/tests/__init__.py`)!